dash/include/dash/matrix/internal/MatrixRefView-inl.h

```diff
@@ -23,28 +23,12 @@ ::MatrixRefView(
   _viewspec(matrix->extents()),
   _l_viewspec()
 {
-  // TODO: Check if initializing local viewspec with default viewspec is okay.
+  // TODO: Check if initializing local viewspec with default viewspec
+  //       is okay.
   DASH_LOG_TRACE_VAR("MatrixRefView(matrix)", matrix);
 }
 
 template<typename T, dim_t NumDim, class PatternT>
-MatrixRefView<T, NumDim, PatternT>
```

Comment:
There always is a copy-constructor if it is not hidden or deleted.
The copy-constructor is implicitly defined by the compiler otherwise.
The previous definition specified it explicitly, but this was only required for logging and prevented default rule of 5 semantics.